src/sentry/constants.py

```diff
@@ -725,7 +725,7 @@ class InsightModules(Enum):
 TARGET_SAMPLE_RATE_DEFAULT = 1.0
 SAMPLING_MODE_DEFAULT = "organization"
 ROLLBACK_ENABLED_DEFAULT = True
-DEFAULT_AUTOFIX_AUTOMATION_TUNING_DEFAULT = "low"
+DEFAULT_AUTOFIX_AUTOMATION_TUNING_DEFAULT = "super_low"
```

Comment:
are these defined on an enum anywhere? might be a good refactor(see https://en.wikipedia.org/wiki/Magic_string). also, why is this needed if you have it in the project option default?

Reply:
Made the refactor

That org setting is to control the default setting for new projects, controlled in the Seer Automation billing page. The per-project setting is the actual setting that controls Seer's behavior.